scripts/checkdeps

```diff
@@ -162,14 +162,14 @@ case "${DISTRO}" in
         [XML::Parser]=XML-Parser
       )
       ;;
-    arch)
+    arch|endeavouros)
       dep_map+=(
         [g++]=g++
         [mkfontscale]=xorg-mkfontscale
         [mkfontdir]=xorg-mkfontdir
         [bdftopcf]=xorg-bdftopcf
         [xsltproc]=libxslt
-        [java]="java-runtime-common jdk8-openjdk"
+        [java]=jdk-openjdk
```

Comment:
this seems to change from a versioned java version to the latest java version, intended? does it still work?

Reply:
I guess https://archlinux.org/packages/extra/x86_64/jdk8-openjdk/ would be fine too - but I've had no issues building Kodi with `jdk-openjdk 18.0.2.u9-1` - anyway the jdk package pulls in the `java-runtime-common` dependency and we can get rid of it